app/views/payments/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= render "header", name: "NEW TRANSACTION" %>
+<%= form_with model: @payment, url: user_payments_path, class: "form" do |f| %>
+  <div class="field form__group first">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", class:"form__input", placeholder: "Transaction name" %>
+  </div>
+  <div class="field form__group">
+    <%= f.label :amount, class: "form-label" %><br>
+    <%= f.number_field :amount, step: :any, class: "form__input", placeholder: "Transaction amount"%>
+  </div>
+  <h2>Category Type: </h2>
+  <div class="field form__group">
+    <%= f.label :utility_id, class: "form-label" %><br>
+    <%= f.collection_select :utility_id, Utility.all.order(:name), :id, :name, class:'select__input' %>
```

Comment:
- [ ] Well done so far setting up the models' association and using has-many-through relationship. However, in the views form doesn't have the ability to choose more than one utility (group) to assign the transaction (payment) to. Please work on this by modifying the form to accept this feature 🙏